packages/react-examples/package.json

```
@@ -61,6 +61,7 @@
     "@fluentui/merge-styles": "^8.0.0-beta.4",
     "@fluentui/react": "^8.0.0-beta.52",
     "@fluentui/react-avatar": "^0.9.1",
+    "@fluentui/react-badge": "*",
```

```suggestion
    "@fluentui/react-badge": "^0.1.0",
```

It should be an exact version,  you can find a proper one in `packages/react-badge/package.json`